publications-2023.bib

```diff
@@ -155,6 +155,19 @@ @Article{2023:orlando:filtering
   publisher = {Elsevier {BV}}
 }
 
+@Article{2023:orlando.benacchio:imex,
+  author    = {Orlando, Giuseppe and Benacchio, Tommaso and Bonaventura, Luca},
+  title     = {An {IMEX-DG} solver for atmospheric dynamics simulations with adaptive mesh refinement},
+  journal   = {Journal of Computational and Applied Mathematics},
+  pages     = {115124},
+  year      = {2023},
+  month     = {aug},
+  volume    = {427},
+  doi       = {10.1016/j.cam.2023.115124},
+  url       = {https://www.sciencedirect.com/science/article/pii/S0377042723000687}  
```

Comment:
```suggestion
  url       = {https://www.sciencedirect.com/science/article/pii/S0377042723000687},
```

Reply:
I should have not properly modified the files. Best